liquibase/src/main/java/org/openmrs/liquibase/SchemaOnlyTuner.java

```diff
@@ -57,6 +58,17 @@ Document replaceBitWithBoolean(Document document) {
 		for (Node node : nodes) {
 			Element parent = node.getParent();
 			parent.addAttribute("type", "BOOLEAN");
+			
+			String defaultValue = parent.attributeValue("defaultValueNumeric");
```

Comment:
Have you looked into why it was originally working without this change?

Reply:
I checked on the 2.6.x and noticed that there it uses the `defaultValueBoolean` however after the update it has changed to `defaultValueNumeric`.

Reply:
Can we look for a way of having it as defaultValueBoolean? Because it makes more sense that way.

Reply:
But boolean is not numeric.

Reply:
That's what this PR is for :)